worker/buildbot_worker/scripts/base.py

```diff
@@ -18,9 +18,9 @@
 from twisted.python import log
 
 
-def isBuildslaveDir(dir):
+def isWorkerDir(dir):
     def print_error(error_message):
-        log.msg("%s\ninvalid buildslave directory '%s'" % (error_message, dir))
+        log.msg("%s\ninvalid worker directory '%s'" % (error_message, dir))
 
     buildbot_tac = os.path.join(dir, "buildbot.tac")
     try:
```

Comment:
should change 
"Application('buildslave')

a few lines after (gh wont allow to comment there)